x-pack/plugins/spaces/index.js

```diff
@@ -90,5 +91,10 @@ export const spaces = (kibana) => new kibana.Plugin({
     initSpacesApi(server);
 
     initSpacesRequestInterceptors(server);
+
+    // Register a function with server to manage the collection of usage stats
+    server.usage.collectorSet.register(getSpacesUsageCollector(server));
+
+    await createDefaultSpace(server);
```

Comment:
not sure how this line is part of the telemetry additions

Reply:
oops, good catch! Will remove